src/lit-element.ts

```diff
@@ -101,7 +118,7 @@ export class LitElement extends PropertiesMixin(HTMLElement) {
    * @param {*} _prevProps Previous element properties
    * @returns {boolean} Default implementation always returns true.
    */
-  protected _shouldPropertiesChange(_props: any, _changedProps: any, _prevProps: any) {
+  _shouldPropertiesChange(_props: any, _changedProps: any, _prevProps: any) {
```

Comment:
Prefer `object` to `any`, here and on `_propertiesChanged`

Reply:
done here and elsewhere